README.md

```diff
@@ -78,6 +78,13 @@ Finally, remember to actually activate the environment when you want to interact
 pipenv shell
 ```
 
+## Creating an iPython Kernel to run your code
+The final step of creating a code-ready environment is to install an iPython kernel that will allow processors such as Jupyter Notebooks to execute code in this envornment.
+Simply run the following inside the pipenv shell:
+```
+python -m ipykernel install --user --name=<name_of_your_repo>
```

Comment:
This is great.  Note that one could also either run `pipenv run init` or `pipenv run create_ipykernel`.  See [here](https://github.com/anmut-consulting/pipenv-cookiecutter/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/scripts/init.sh) and the [Pipfile](https://github.com/anmut-consulting/pipenv-cookiecutter/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/Pipfile) under [scripts]

Reply:
@aj-cloete thank you for checking this, just a quick one- I did run pipenv run init but it didn't generate an ipykernel for me, is it supposed to? If so, perhaps it's an update that I don't have on my version....

Reply:
Yes, this is a problem I still need to figure out: how do we make sure that updates to this repo also get pushed to repos where it already used the cookiecutter?  Also, how do we know when it is out of date...I think we may need to think about cookiecutter versioning within the generated repos.